src/Home/Home.tsx

```diff
@@ -1,5 +1,80 @@
 import * as React from 'react';
+import glamorous from 'glamorous';
+import './Home.css';
+const { Div } = glamorous;
+import PageCard from './PageCard';
+import CircleButton from './CircleButton';
 
-const Home = () => <h1>Home Page</h1>;
+class Home extends React.Component {
+  render() {
+    return (
+      <Div margin="5px auto" padding="20px" width="100%">
+        <Div id="nav" display="flex">
+          <h1 id="titleText"> panel.uclaradio.com: links </h1>
+          <Div
+            className="buttonGroup"
+            display="grid"
+            grid-template-columns="5vw 5vw 5vw"
+          >
+            <CircleButton />
+            <CircleButton />
+            <CircleButton />
+          </Div>
+        </Div>
+
+        <Div id="search" display="flex" justifyContent="center">
+          <Div id="searchPadding" width={600} padding="30px">
+            <input placeholder="Search" type="text" id="searchBar" />
+          </Div>
+        </Div>
+
+        <Div
+          className="link-grid"
+          display="grid"
+          grid-template-columns="repeat(auto-fill,minmax(400px, 1fr))"
+          grid-template-rows="repeat(9, minmax(400px, 1fr))"
+          grid-column-gap="30px"
+          padding="50px"
+        >
+          {/* {
```

Comment:
Let's remove this comment for now, we'll come back to dynamically populating the page soon.